Makefile

```diff
@@ -21,7 +21,7 @@ run: generate fmt vet
 
 # Install CRDs into a cluster
 install: manifests
-	kubectl apply -f vendor/github.com/openshift/cluster-api/config/crds/
+	kubectl apply -f vendor/github.com/openshift/machine-api-operator/install
```

Comment:
What CRD(s) in the machine-api-operator does the CAPBM need to function?

Reply:
The definitions of 'machine', 'machinesets', etc come from MAO (and formerly ClusterAPI). Thanks.